traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/CatalogItemDetailsFragment.java

```diff
@@ -29,11 +28,12 @@
 import com.guestlogix.traveleruikit.widgets.WrapContentViewPager;
 
 public class CatalogItemDetailsFragment extends Fragment {
-    static public String ARG_CATALOG_ITEM_DETAILS = "ARG_CATALOG_ITEM_DETAILS";
-    static private String TAG = "CatalogItemDetailsFragment";
+    private static final String ARG_CATALOG_ITEM_DETAILS = "ARG_CATALOG_ITEM_DETAILS";
+    private static final String TAG = "CatalogItemDetailsFrag";
```

Comment:
no short form..complete class name

Reply:
Its length broke the logcat 23 character limit. I believe logcats sent with such a tag get dropped. It would not be an issue >= api 24